packages/platform-ios/src/commands/runIOS/index.js

```diff
@@ -418,15 +425,17 @@ function printFoundDevices(devices) {
   return output;
 }
 
-function getProcessOptions(launchPackager, port) {
+function getProcessOptions(launchPackager, port, terminal) {
+  const env = {...process.env, RCT_TERMINAL: terminal};
```

Comment:
Any reason to do spreading twice? I would leave it as is and just add `RCT_TERMINAL: terminal` to line 433.

Reply:
```suggestion
```

Reply:
Just didn't want to specify `RCT_TERMINAL` twice, I'll change it though.